WireUI/Sources/WireUIFoundation/Main/Models/MainTabBarControllerContent.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+/// The type of tabs of the main tab bar controller shows.
+/// Since UIKit has a type `UITab` this type has been suffixed with "Content".
+public enum MainTabBarControllerContent: Int, CaseIterable {
+    case contacts // will be removed in navigation overhaul
+    case conversations
+    case folders // will be removed in navigation overhaul
+    case archive, settings
```

Comment:
Any reason settings is on the same line here?

Reply:
For navigation overhaul it will be `case conversations, archive, settings`.
For `develop` I had to inject the other two cases and left a comment.
Anyway, it will be gone soon, hopefully.